code/datums/shuttles.dm

```diff
@@ -0,0 +1,106 @@
+/datum/map_template/shuttle
+	name = "Base Shuttle Template"
+	var/prefix = "_maps/map_files/shuttles/"
+	var/suffix
+	var/port_id
+	var/shuttle_id
+
+	var/description
+	var/admin_notes
+
+/datum/map_template/shuttle/New()
+	shuttle_id = "[port_id]_[suffix]"
+	mappath = "[prefix][shuttle_id].dmm"
+	. = ..()
+
+/datum/map_template/shuttle/emergency
+	port_id = "emergency"
+	name = "Base Shuttle Template (Emergency)"
+
+/datum/map_template/shuttle/cargo
+	port_id = "cargo"
+	name = "Base Shuttle Template (Cargo)"
+
+/datum/map_template/shuttle/ferry
+	port_id = "ferry"
+	name = "Base Shuttle Template (Ferry)"
+
```

Comment:
yeah it is on tgs template list..i actually have a few there i haven't ported over but as this deals with ferrys/shuttles should i add it this go around?